tests/kafkatest/tests/core/security_rolling_upgrade_test.py

```diff
@@ -126,10 +126,10 @@ def remove_separate_broker_listener(self, client_security_protocol):
         self.bounce()
 
     @cluster(num_nodes=8)
-    @matrix(client_protocol=[SecurityConfig.SSL])
+    @matrix(client_protocol=[SecurityConfig.SSL], use_new_coordinator=[True, False])
```

Comment:
Do all the tests in this file use ZK or KRaft? I suppose that it is ZK by default...

Reply:
oh yes I must've missed to check in this test, yes zk is the default and since kraft wasn't passed as the metadata quorum ig all of the tests use zk and mentioning the group coordinator type is redundant, I'll remove it, thanks for the catch!